DataConnectors/Agari/azuredeploy_Agari_API_FunctionApp.json

```diff
@@ -0,0 +1,301 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "FunctionName": {
+            "defaultValue": "agari",
+            "type": "string"
+        },
+        "clientID": {
+            "type": "string",
+            "defaultValue": "Enter Agari Client ID"
+        },  
+        "clientSecret": {
+            "type": "string",
+            "defaultValue": "Enter Agari Client Secret"
+        },  
+        "workspaceID": {
+            "type": "string",
+            "defaultValue": "<workspaceID>"
+        },
+        "workspaceKey": {
+            "type": "string",
+            "defaultValue": "<workspaceKey>"
+        },
+		"enableBrandProtectionAPI": {
+            "type": "bool"
+		},
+		"enablePhishingDefenseAPI": {
+            "type": "bool"
+        },
+        "enablePhishingResponseAPI": {
+            "type": "bool"
+        },
+		"enableSecurityGraphSharing": {
+            "type": "bool"
+		},
+	 	"GraphTenantID": {
+            "type": "string",
+            "defaultValue": "Enter MS Graph Tenant ID"
+        },  
+        "GraphClientID": {
+            "type": "string",
+            "defaultValue": "Enter MS Graph Client ID"
+        },  
+		"GraphClientSecret": {
+            "type": "string",
+            "defaultValue": "Enter MS Graph Client Secret"
+        }  
+
+    },
+    "variables": {
```

Comment:
If we use the variable, I'm assuming I need to remove it from the start of the list of parameters (lines 5-8). Will I still be be able to access the name of application through "functionName": "[parameters('FunctionName')]", (line 239)?

Reply:
You still need to expose the fun name as parameter and in case customer needs to change it, but use it as seed here to generate unique names. 

Please take a look of current function connectors, they all follow the same pattern. We will also update our template to reflect the change: https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/CiscoUmbrella/azuredeploy_CiscoUmbrella_API_FunctionApp.json

https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/VMware%20Carbon%20Black/azuredeploy_VMwareCarbonBlack_API_FunctionApp.json